packages/babel-preset-udemy-website/tests/babel-polyfill.js

```diff
@@ -0,0 +1 @@
+import '@babel/polyfill';
```

Comment:
If I don't use a separate file to pull in @babel/polyfill, then about 30 import lines get added to each of the resulting files.

Reply:
Hmm, I actually think it would be better if we explicitly saw those 30 import lines at our test results so that we'd get notified if something changes. So I think we should directly import `import '@babel/polyfill';`